pkg/restore/restore_crd.go

```diff
@@ -0,0 +1,237 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package restore
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/debug"
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var getCrName = `
+	kubectl -n %s get %s -o json | jq -r '.items[] | select(.metadata.deletionGracePeriodSeconds == 0) | .metadata.name'
```

Comment:
Does the plugin know if it's running in the context of `oc` or `kubectl`? We would need to use the same tool.

Reply:
currently, the tool is doesn't have the capability of finding if it `kubectl` or `oc`. But I think it will be better if in the future if the tool is able to find the differences.